pnpm-workspace.yaml

```diff
@@ -0,0 +1,40 @@
+packages:
+    - "config/build"
+    - "dev"
+    - "packages/*"
+    - "vendor/*"
+
+catalog:
```

Comment:
The magic of `pnpm` catalogs.  We can define the version of each peer dependency here and then update _all_ of our Perseus packages to the same version in one fell swoop!

Reply:
Do we also need to update `utils/sync-dependencies.ts`?

Reply:
Oh yes! Thank-you for thinking of that! I'll update this PR.